fixieai/agents/code_shot.py

```diff
@@ -121,6 +127,7 @@ def __init__(
         conversational: bool = False,
         oauth_params: Optional[oauth.OAuthParams] = None,
         llm_settings: Optional[LlmSettings] = None,
+        allowed_agent_id: Optional[str] = None,
```

Comment:
The naming of this seems strange, should this just be `agent_id`?

Reply:
The intent was to try to make it explicitly narrow to both minimize dependencies and avoid users spending time setting this. (And also I didn't want to make it a breaking change.)

I really only added the argument for tests, but I could also just temporarily set the environment variable in the test to take it out of the API entirely?

Reply:
I took it out of the API to try to remove it from the user's hands entirely -- lmk if you think that makes sense.